src/documentation/0056-node-control-flow/index.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Asynchronous flow control
+description: 'JavaScript control flow is all about callback handling. Here are strategies to help your development.'
+authors: aug2uag
+section: Getting Started
+---
+
+I'm presenting material that's helped me greatly in the past from [Mixu's Node.js Book](http://book.mixu.net/node/ch7.html).
```

Comment:
Is this a copy paste of the material from that book?

If so we probably can't land it.

If it is originally copy we likely want to modify this to not use `I'm`, as the docs are generally not written in the first person.

```suggestion
The material in this post has been heavily inspired by [Mixu's Node.js Book chapter 7](http://book.mixu.net/node/ch7.html).

Reply:
sorry i had missed your suggestion, made the update just now